src/teams/teams.service.ts

```diff
@@ -0,0 +1,49 @@
+import { Injectable } from '@nestjs/common';
+import { Repository } from 'typeorm';
+import { Team } from './team.entity';
+import { InjectRepository } from '@nestjs/typeorm';
+import { CreateTeamDto, TeamDto, UpdateTeamDto } from './dto';
+import { TeamNotFoundError } from './errors/team-not-found.error';
```

Comment:
This is wrong, we need an index file at the error folder, similar to how you did the dtos.